vignettes/tech-dm-join.Rmd

```diff
@@ -96,11 +96,11 @@ The difference is the `name` column from the `airlines` table.
 
 ```{r}
 dm %>%
-  tbl("flights") %>%
+  .[["flights"]] %>%
```

Comment:
I'd prefer `dm$flights` here, without pipe.

Reply:
sure - a commit is coming